druid-shell/src/platform/windows/window.rs

```diff
@@ -212,6 +229,12 @@ const DS_RUN_IDLE: UINT = WM_USER;
 /// time it is handled, we can successfully borrow the handler.
 pub(crate) const DS_REQUEST_DESTROY: UINT = WM_USER + 1;
 
+/// Message relaying a request to handle dropped messages.
```

Comment:
I'd rework the name and comment. First, messages aren't dropped if they're placed in the queue. We use the word "blocked" and that's ok, but I think "deferred" would actually be best. Here's my suggestion for the comment:

/// A message which must be delivered deferred due to reentrancy.
///
/// Due to the architecture of Windows, it sometimes delivers messages
/// reentrantly. This is problematic when the mutable state for the window
/// is borrowed, as it's not possible to safely dispatch the message. The two
/// choices are to drop the message, or put it in a queue for deferred
/// processing.
///
/// This mechanism should be used very sparingly, as delivery of these
/// deferred messages can not be guaranteed in a timely and reliable
/// fashion. In particular, we do not run our own runloop when handling
/// live resize, when a modal dialog is open, or when the application is a
/// a guest (such as a VST plugin).